request_windows.go

```diff
@@ -14,7 +14,15 @@ func testOsSys(sys interface{}) error {
 	return nil
 }
 
-func toLocalPath(p string) string {
+func toLocalPath(workDir, p string) string {
+	if workDir != "" {
+		if !filepath.IsAbs(p) && !path.IsAbs(p) {
```

Comment:
The code `lp := filepath.FromSlash` below demonstrates that the `p` argument should always be compliant with `path.Join` here.

Reply:
I understand that's the intent, but it doesn't give any guarantees for the input. `FromSlash` is pretty much just no-op on a backslash-based path. And if we try to `path.Join` with a unix-y workDir, then we're in for trouble.

Reply:
`p` should never be non-unixy. This is a requirement of the standard.